hadoop-tools/hadoop-federation-balance/src/site/markdown/FederationBalance.md

```diff
@@ -0,0 +1,156 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+Federation Balance Guide
+=====================
+
+---
+
+ - [Overview](#Overview)
```

Comment:
Isn't toc already generating with some command?

Reply:
Thanks your comments ! I did a test. If I removed the toc then there would be no toc in the generated html.

Reply:
Take a look at:
https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/ShortCircuitLocalReads.html
It just has:
<!-- MACRO{toc|fromDepth=0|toDepth=3} -->

Reply:
Check:
https://raw.githubusercontent.com/apache/hadoop/a55d6bba71c81c1c4e9d8cd11f55c78f10a548b0/hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/ShortCircuitLocalReads.md